httpx/_transports/wsgi.py

```diff
@@ -1,14 +1,38 @@
 import io
 import itertools
 import sys
+import types
 import typing
 
+import typing_extensions as te
```

Comment:
I can't see where this is coming from in the `requirements.txt`... is this built-in or a package?

Also, can we just `import typing_extensions` rather than `import typing_extensions as te`?